ui/packages/consul-peerings/app/components/consul/peer/form/generate/index.hbs

```diff
@@ -17,7 +17,20 @@
       id={{id}}
     >
 
-      <fsm.State @matches={{array 'idle'}}>
+      <fsm.State @matches={{array 'idle' 'error'}}>
+        {{#if fsm.state.context.error}}
```

Comment:
May I suggest to, instead of relying on the context here, to add another `<fsm.State @matches={{array 'error'}}>`-block? That way, we will display the notice based on the state of the state-machine, not based on a property set on `context`.

Reply:
Oooh yes! I like that much better, I added a commit on the end here